examples/README.md

```diff
@@ -3,6 +3,10 @@ Optuna Examples
 
 This page contains a list of example codes written with Optuna.
 
+You can use our docker images with the tag ending with `-dev` to run most of the examples.
+For example, you can run [PyTorch Lightning Simple](./pytorch_ignite_simple.py) via `docker run --rm optuna/optuna:py3.7-dev python pytorch_lightning_simple.py`.  
+Also, you can try our visualization example in Jupyter Notebook by opening `localhost:8888` in your browser after executing `docker run -p 8888:8888 --rm optuna/optuna:py3.7-dev jupyter notebook --allow-root --no-browser --port 8888 --ip 0.0.0.0 --NotebookApp.token='' --NotebookApp.password=''`.
```

Comment:
We may want to shorten this `docker run` command by configuring Jupyter in Dockerfile. What's your take?

Reply:
I'm fine either, but how about inserting a new line between texts and a command like this? This may help the readability for users. What do you think?
```suggestion
Also, you can try our visualization example in Jupyter Notebook by opening `localhost:8888` in your browser after executing this:
```bash
docker run -p 8888:8888 --rm optuna/optuna:py3.7-dev jupyter notebook --allow-root --no-browser --port 8888 --ip 0.0.0.0 --NotebookApp.token='' --NotebookApp.password=''
```

Reply:
Ops, closing backticks are missing?